### PR TITLE
Add component tests

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Add missing component tests.

#### What problem is this solving?
Lack of tests for the `CategoryMenu` component.

#### How should this be manually tested?
Run `yarn test` or `npm t` inside the `react` folder.

#### Screenshots or example usage
N/A

### CODE REVIEW

File: react/__tests__/CategoryMenu.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  it('should be rendered', () => {
+    expect(wrapper).toBeDefined()
+  })
+
+  it('should match snapshot', () => {
+    expect(wrapper.container).toMatchSnapshot()
+  })
+
+  it('should render 3 menu items', () => {
```
Comment (andreldsa) [inline react/__tests__/CategoryMenu.test.js]: You should create failed cases to ensure that the result of the positive it's correct.

File: react/components/CategoryItem.js [hunk 2]
```diff
@@ -1,77 +1,69 @@
...
-              </div>
-            )}
-          </div>
-        </LoadingBar>
+          {hasChildren && (
+            <div className="vtex-category-sub-menu pv6 ph5 br2 br--bottom">
+              <ul className="list ma0 pa0 f6">
+                {category.children.map(subCategory => (
+                  <li key={subCategory.id} className="lh-copy">
+                    <a
```
Comment (leticiawanderley) [inline react/components/CategoryItem.js]: You can try to test the click events on a category item.

File: react/package.json [hunk 3]
```diff
@@ -1,8 +1,35 @@
 {
   "name": "vtex.render-getting-started",
   "version": "0.0.0",
+  "scripts": {
+    "test": "jest",
+    "update-snapshots": "jest --updateSnapshot"
```
Comment (leticiawanderley) [inline react/package.json]: I suppose you could remove this script, the user can update the snapshots using `npm test -- -u`

**Comment by lucasecdb** [comment]
I couldn't test the redirect with on click on the category label, because I'm using a simple `<a></a>` tag, and not calling anything like `window.location.assign`. Other thing I couldn't test was the dropdown opening on hover, because the stylesheet for the components is loaded by the pages framework.

File: react/__tests__/CategoryItem.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
```
Comment (klzns) [inline react/__tests__/CategoryItem.test.js]: ⛏ Vc pode mudar a configuracao do eslint pra aceitar `jest` como global.

**Pull request merged**